test/evergreen/code_change_report/code_change_info.py

```diff
@@ -140,22 +295,59 @@ def create_report_info(change_list: dict, coverage_data: dict):
                 line_info['old_lineno'] = line.old_lineno
 
                 if line.new_lineno > 0:
-                    line_info['count'] = find_line_coverage(coverage_data=coverage_data, file_path=new_file, line_number=line.new_lineno)
-                    line_info['branches'] = find_covered_branches(coverage_data=coverage_data, file_path=new_file, line_number=line.new_lineno)
+                    line_coverage = find_line_coverage(coverage_data=coverage_data,
+                                                       file_path=new_file,
+                                                       line_number=line.new_lineno)
+                    branch_coverage = find_covered_branches(coverage_data=coverage_data,
+                                                            file_path=new_file,
+                                                            line_number=line.new_lineno)
+                    line_info['count'] = line_coverage
+                    line_info['branches'] = branch_coverage
+                    if line.old_lineno < 0:
```

Comment:
This is an artefact of format of the diff information. If a line is simply moved (eg because something is inserted or deleted before it) then it will have an 'old' line number. On the other hand, if a line is changed, then you see it as a line being deleted, and a line being added. The new version of the line will not have an 'old' line number. You can see this happening in this Code Change Report at line 850 in conn_log.c: https://build_external.s3.amazonaws.com/wiredtiger/code-statistics/caa512a029b5486c94fc6eb8a0070c66d1948558/code-change-report_wiredtiger_code_statistics_caa512a029b5486c94fc6eb8a0070c66d1948558_24_02_28_22_21_56-0/code_change_report.html
I'll add more comments to make that clearer.

Reply:
Done.